code/go/0chain.net/smartcontract/storagesc/challenge.go

```diff
@@ -972,57 +973,82 @@ func (sc *StorageSmartContract) populateGenerateChallenge(
 		return nil, errors.New("invalid blobber for allocation")
 	}
 
+	if alloc.DataShards+1 > minValidators {
```

Comment:
looks like we should get fixed number of validators
```yaml
    # number of validators per challenge
    validators_per_challenge: 2
```

Reply:
I think it's not a fixed number of validators, but rather a lower bound i.e. [minimum number of validators](https://github.com/0chain/0chain/blob/c51117b529af747139ed591dacc4d3f08bde1aa0/code/go/0chain.net/smartcontract/storagesc/challenge.go#L1107)

Reply:
this doesn't make sense
```
minValidators = alloc.DataShards + 1
```
we only need to get the minimum validators for any challenge, it's due to @guruhubb